HappyScript_MAIN.r

```diff
@@ -0,0 +1,10 @@
+
+source(file.path("1_fetch","src","fetch.r"))
+source(file.path("2_process","src","process.r"))
+source(file.path("3_visualize","src","plot.r"))
```

Comment:
I like your approach of building a HappyScript_MAIN.r that's the main orchestrator of the pipeline, and I think you'll find nice alignment with this approach and with the *targets* approach you're about to try. 

One note here, not because there's something to fix but just an FYI in case you prefer it, is that you _can_ use forward slashes to build a file path, e.g., `source('3_visualize/src/plot.r')`. Nothing wrong with the `file.path` approach except that it takes more typing.

Reply:
Thanks for the suggestion.